ts/Stock/Indicators/VBP/VBPIndicator.ts

```diff
@@ -46,6 +46,7 @@ const {
 } = SeriesRegistry.seriesTypes;
 import U from '../../../Core/Utilities.js';
 import StockChart from '../../../Core/Chart/StockChart.js';
+import DataExtremesObject from '../../../Core/Series/DataExtremesObject';
```

Comment:
Import type only

Reply:
Oh, I must have missed this one - thanks!